css/src/new-redirects.css

```diff
@@ -61,6 +61,22 @@ body.seo_page_wpseo_redirects {
 	}
 
 	.yst-root {
+		.yst-notifications {
+			max-height: calc(100% - 4rem - 32px); /* Extra 32px for the height of the WP admin bar. */
```

Comment:
Please check if you can use tailwind css for that https://tailwindcss.com/docs/max-height#examples

Reply:
I would like to have a patch to test it. Lets see if we can find a cleaner way to address that.